test/e2e/create_test.go

```diff
@@ -175,17 +175,68 @@ func (tc *testContext) testMachineConfiguration(t *testing.T) {
 	}
 	_, err := tc.createWindowsMachineSet(gc.numberOfMachineNodes, false)
 	require.NoError(t, err, "failed to create Windows MachineSet")
+	machineCreationTime := time.Now()
 
 	t.Run("Machine configuration while private key change", tc.testMachineConfigurationWhilePrivateKeyChange)
 
 	machines, err := tc.waitForWindowsMachines(int(gc.numberOfMachineNodes), "Provisioned", false)
 	require.NoError(t, err, "error waiting for Windows Machines to be provisioned")
+	machineProvisionedTime := time.Now()
```

Comment:
For my own education, what is the default time zone? Should it specific one, ie. `time.Now().UTC()` ?